<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,2,0,1" columnstretch="1,1,1">
      <item row="3" column="1">
       <widget class="QPushButton" name="trade_btn">
        <property name="text">
         <string>매매시작</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="infinite_r_btn">
        <property name="text">
         <string>무한매수</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="couple_r_btn">
        <property name="text">
         <string>커플링 매매</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QTableWidget" name="list_view"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="stop_btn">
        <property name="text">
         <string>매매정지</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="asset_info"/>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QTextBrowser" name="log_view"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="profit_lineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="coin_comboBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
